docs/spec_draft.md

```diff
@@ -195,7 +195,9 @@ Performs element-wise addition of two tensors `lhs` and `rhs` and produces a
 unsigned/signed overflow/underflow, the result is implementation-defined and one
 of the following:
 
-  * mathematical result modulo $2^n$, where n is the bit width of the result.
+  * mathematical result modulo $2^n$, where n is the bit width of the result,
+  for unsigned overflow/underflow. For signed interger overlfow/underlow, wraps
+  the result around the representatble range $[2^{n-1} - 1$, $-2^{n-1}]$.
```

Comment:
Let's change the range to start from a negative value and end with a positive value?